test/integration/gateways/securion_pay_test.exs

```diff
@@ -0,0 +1,78 @@
+defmodule Gringotts.Integration.Gateways.SecurionPayTest do
+  use ExUnit.Case, async: true
+  use ExVCR.Mock, adapter: ExVCR.Adapter.Hackney
+
+  alias Gringotts.{
+    CreditCard
+  }
+
+  alias Gringotts.Gateways.SecurionPay, as: Gateway
+
+  @moduletag integration: false
+
+  @amount Money.new(42, :EUR)
+
+  @bad_card %CreditCard{
+    first_name: "Harry",
+    last_name: "Potter",
+    number: "4200000000000000",
+    year: 2009,
+    month: 12,
+    verification_code: "123",
+    brand: "VISA"
+  }
+
+  @good_card %CreditCard{
+    first_name: "Harry",
+    last_name: "Potter",
+    number: "4200000000000000",
+    year: 2019,
+    month: 12,
+    verification_code: "123",
+    brand: "VISA"
+  }
+
+  @opts [
+    config: [secret_key: "sk_test_Ff2Jpq59sSWT7qmI9atii5zR"],
+    customer_id: "cust_zpYEBK396q3rvIBZYc3PIDwT"
```

Comment:
Again, replace with dummy secret key and customer_id.

Reply:
If I replace it with dummy keys my test will fail. Should I change it before pushing it to remote?

Reply:
The tests will definitely need a valid secret key, use the secret key:
1. that is displayed publicly in SecurionPay docs (if any)
2. the [one used in](https://github.com/activemerchant/active_merchant/blob/master/test/fixtures.yml#L1096) activemerchant [tests](https://github.com/activemerchant/active_merchant/blob/master/test/remote/gateways/remote_securion_pay_test.rb) (if any, and that it works)
3. (worst case) that you own.